include/silo/common/association_list.h

```diff
@@ -0,0 +1,25 @@
+#include <algorithm>
+#include <optional>
+#include <vector>
+
+template <typename K, typename V>
+class AssociationList {
```

Comment:
I don't recall - did we agree that we keep this class?

Reply:
Alex didn't like the idea so I removed it.